modules/apps/portal-store/portal-store-s3/src/main/java/com/liferay/portal/store/s3/AbortedMultipartUploadCleanerMessageListener.java

```diff
@@ -70,6 +71,12 @@ protected void deactivate() {
 
 	@Override
 	protected void doReceive(Message message) throws Exception {
+		String storeClassName = PropsValues.DL_STORE_IMPL;
```

Comment:
@yuhai Please use portal profile, do not enable this component when dl store is not set, like what you did for the company log servlet.